src/BenchmarkDotNet/Loggers/TextLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IO;
+
+namespace BenchmarkDotNet.Loggers
+{
+    public class TextLogger : ILogger, IDisposable
+    {
+        private readonly TextWriter writer;
+
+        public TextLogger(TextWriter writer) => this.writer = writer;
+
+        public void Dispose() => writer.Dispose();
+
+        public string Id => nameof(TextLogger);
```

Comment:
Please make `Id` virtual and keep `nameof(StreamLogger)`.

Reply:
my bad, should be fixed now